peer/main_test.go

```diff
@@ -39,5 +39,5 @@ func TestSystem(t *testing.T) {
 	// if *systemTest {
 	// 	main()
 	// }
-	main()
+	//main()
```

Comment:
This is "dead" code (commented). Should we preserve dead code? (Git implicitly preserves it in the history)